src/app.js

```diff
@@ -31,15 +31,14 @@ server.get("/", (req, res) => {
   });
 });
 
-
-
-
-
 server.use(morgan("dev"));
 server.use(cors());
+server.use(cookieParser())
 
 server.use("/api-docs", swaggerUi.serve, swaggerUi.setup(swaggerDoc, { explorer: true }));
-server.use("*", (req, res, next) => {
-	res.status(404).json({ error: "NOT FOUND", });
-});
+// server.use("*", (req, res, next) => {
+// 	res.status(404).json({ error: "NOT FOUND", });
+// });
+server.use("/profile",userRoute);
```

Comment:
Do not put this here directly! Please use the file `/routers/index.js` to hold all of routes.